### DISCUSSION TITLE
Language Switcher

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#697] [Q&A (Help)] [answered]
In repo https://github.com/laravel-backpack/language-switcher i read:
Try it right now, in [our online demo](https://demo.backpackforlaravel.com/admin/).

But where is it in your demo ?? :)

other, can use it also on login and registration pages ?

**Comment by jcastroa87** [comment, marked as answer]
Hello @blondie63 we are working on it, soon will be available for public demo.

Cheers.

**Comment by promatik** [comment]
Hey @blondie63 the PR for the demo is ready, it will soon be merged 👌
https://github.com/Laravel-Backpack/demo/pull/576

> _can use it also on login and registration pages?_

Yes!
The component can be used in the login/register forms.
It actually can be used anywhere you want, for it to work in other places than the backpack "scope", like if you want to use the component in your web pages, you'll need to add the middleware to the respective middleware groups (most probably web middleware), but you can also use it in an api, by adding it to the api middleware 🤷‍♂️
